test.sh

```diff
@@ -12,7 +12,7 @@ cd /tmp/
 wget -nv https://dl.google.com/go/go1.11.linux-amd64.tar.gz
 wget -nv https://dl.google.com/go/go1.10.4.linux-amd64.tar.gz
 wget -nv https://dl.google.com/go/go1.9.7.linux-amd64.tar.gz
-wget -nv https://dl.google.com/go/go1.8.7.linux-amd64.tar.gz
+# wget -nv https://dl.google.com/go/go1.8.7.linux-amd64.tar.gz
```

Comment:
please keep go1.8. I hope to consider whether we should remove support go1.8 .

Reply:
@mattn 
I figured you were still considering supporting version 1.8 or not. But until then, I commented it out. Did not see any reason to download the file when it is not used. If oci8 becomes usable with Go 1.8, we can comment it back in, along with the other comments for 1.8.